apps/wallet-dashboard/components/migration/resolved-migration-objects/ResolvedObjectList.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { ExpirationObjectListEntries, ResolvedObjectsList } from '@/lib/types';
+import { getObjectListReactKey, isNftObjectList, isObjectTypeBasic } from '../helpers';
+import { ResolvedObjectCard } from './ResolvedObjectCard';
+
+interface ObjectsListProps {
+    objects: ExpirationObjectListEntries;
+    isTimelockedObjects: boolean;
+}
+
+export function ResolvedMigrationObjectList({ objects, isTimelockedObjects }: ObjectsListProps) {
```

Comment:
maybe this could be a virtual list, just in case the list gets too large for some users :pray: